src/labelle/lib/devices/dymo_labeler.py

```diff
@@ -274,7 +273,10 @@ def __init__(
 
     @property
     def height_px(self):
-        return DymoLabelerFunctions.height_px(self.tape_size_mm)
+        """Get the tape size in pixels."""
+        return self._deviceConfig.get_tape_print_size_and_margins_px(self.tape_size_mm)[
+            0
+        ]
```

Comment:
Using `[0]` is rather odd. Please refer to my earlier comment about using properties.

Reply:
I want the first item in the tuple. Since it has no name how do you refer to it?

Reply:
Made it a named tuple and used name